src/main/resources/db/migration/V2__dml.sql

```diff
@@ -135,3 +200,10 @@ values ('22afc9b4-807f-4eb2-b286-788631d1ed55', current_date, 'admin@gelecekbili
         'MALE', 'Gelecek', 'Bilimde',
         '$2a$10$atVE.cT5YpEOS7ZLSoVdk.QKVyYBCgvNhvQEuCcXbEMpohYIjbZDG', 'VERIFIED',
         'c147b5c2-87f7-4bb7-a165-368f639d8c3c', 'gelecekbilimde', current_timestamp);
+
+insert into gb_user (id, birth_date, email, gender, first_name, last_name, password,
+                     status, role_id, created_by, created_at)
+values ('d22e5b1f-d01e-4987-baf8-ed29ec4fa6cf', current_date, 'user@gelecekbilimde.net',
+        'MALE', 'Gelecek', 'Bilimde',
+        '$2a$10$atVE.cT5YpEOS7ZLSoVdk.QKVyYBCgvNhvQEuCcXbEMpohYIjbZDG', 'VERIFIED',
+        'e3a1a32d-fcd7-46f0-bb2b-201df6b2b808', 'gelecekbilimde', current_timestamp);
```

Comment:
@Coderabbitai Since all the data in this file is our test data, we do not think there is any security vulnerability here. When we take the application to production, we will update all the data. So don't make any suggestions.